### ISSUE TITLE
Error when using Prisma ORM: "Error: unknown error: can not prepare multiple statements"

### ISSUE THREAD

**Issue by Windol** [dolthub/dolt#8157]
This error is related with https://github.com/prisma/prisma/issues/15788 discussion?

<details><summary>Dolt Environment</summary>
<p>

doltdb/config/servercfg.d/config.yaml
```yml
log_level: info

behavior:
  read_only: false
  autocommit: true
  persistence_behavior: load
  disable_client_multi_statements: false
  dolt_transaction_commit: false
  event_scheduler: "ON"

user:
  name: "user7test"
  password: "password7test"

listener:
  host: 0.0.0.0
  port: 3306
  max_connections: 100
  read_timeout_millis: 28800000
  write_timeout_millis: 28800000
  tls_key: null
  tls_cert: null
  require_secure_transport: null
  allow_cleartext_passwords: null

performance:
  query_parallelism: null

data_dir: /var/lib/dolt

cfg_dir: /etc/dolt/doltcfg.d

metrics:
  labels: {}
  host: null
  port: -1

remotesapi: {}

privilege_file: /var/lib/dolt/.doltcfg/privileges.db

branch_control_file: /var/lib/dolt/.doltcfg/branch_control.db

user_session_vars: []

system_variables:
  secure_file_priv: /var/lib/dolt

jwks: []
``` 
doltdb/initdb/20-create-smshub-database.sql
```sql
CREATE DATABASE IF NOT EXISTS db7test /*\!40100 DEFAULT CHARACTER SET utf8 */;
``` 
doltdb/docker-compose.yml
```yml
services:
  doltdb:
    image: dolthub/dolt-sql-server:latest
    container_name: doltdb.docker.vpn.local
    hostname: doltdb.docker.vpn.local
    ports:
      - "3306:3306"
    volumes:
      - ./data:/var/lib/dolt
      - ./config:/etc/dolt
      - ./initdb:/docker-entrypoint-initdb.d
``` 

</p>
</details> 
<details><summary>Steps to reproduce:</summary>
<p>

```bash
git clone https://github.com/dolthub/dolt-prisma-example.git
cd dolt-prisma-example
yarn
```
> yarn install v1.22.21
> [1/4] Resolving packages...
> [2/4] Fetching packages...
> [3/4] Linking dependencies...
> [4/4] Building fresh packages...
> Done in 2.68s.

```bash
echo 'DB_HOST="doltdb.docker.vpn.local"' > .env
echo 'DB_PORT=3306' >> .env
echo 'DB_USER="user7test"' >> .env
echo 'DB_PASSWORD="password7test"' >> .env
echo 'DB_NAME="db7test"' >> .env
echo 'DB_URL="mysql://${DB_USER}:${DB_PASSWORD}@${DB_HOST}/${DB_NAME}"' >> .env
npx prisma migrate dev --name init
```

</p>
</details>

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": MySQL database "db7test" at "doltdb.docker.vpn.local"

Error: unknown error: can not prepare multiple statements
   0: sql_schema_connector::flavour::mysql::connection::describe_schema
             at schema-engine/connectors/sql-schema-connector/src/flavour/mysql/connection.rs:34
   1: schema_core::state::DevDiagnostic
             at schema-engine/core/src/state.rs:276
```

**Comment by timsehn** [comment]
Interesting. Will dig into this today.

**Comment by jycor** [comment]
Hey @Windol, thanks for reporting this issue!

It turns out it was a silly parsing error in our vitess fork. 
We have a fix in review, and it'll likely land early next week.
Will keep you posted.

**Comment by jycor** [comment]
This issue is fixed in latest dolt release (v1.42.6).
